fdbserver/storageserver.actor.cpp

```diff
@@ -1948,7 +1948,7 @@ void splitMutation(StorageServer* data, KeyRangeMap<T>& map, MutationRef const&
 ACTOR Future<Void> logFetchKeysWarning(AddingShard* shard) {
 	state double startTime = now();
 	loop {
-		wait(delay(600));
+		wait(delay(g_network->isSimulated() ? 5 : 600));
```

Comment:
Can we make this a randomly generated value in a range of [2, 50]?
It may increase the coverage and make it easier to catch the problem.